src/components/Note.tsx

```diff
@@ -18,14 +19,15 @@ export const Note: React.FC<Props> = (props) => {
   if (note === null) return null;
   const photoUrl = PersonHelper.getPhotoUrl(note.person);
   var datePosted = new Date(note.dateAdded);
-  datePosted.setTime(
-    datePosted.getTime() - datePosted.getTimezoneOffset() * 60 * 1000
-  );
+  // datePosted.setTime(
+  //   datePosted.getTime() - (datePosted.getTimezoneOffset() * 60 * 1000)
+  // );
```

Comment:
@jzongker, so I've chosen to comment this part as it was resulting in negative time. Besides each note we show time about when it was created / updated, that date came as negative at times due to offset.

Let me know what you think.